x/auth/client/cli/sign.go

```diff
@@ -5,6 +5,8 @@ import (
 	"os"
 	"strings"
 
+	"github.com/pkg/errors"
```

Comment:
Unless you need to use the `Wrap()` function, this could be replace by the standard package errors:

```suggestion
	"errors"
```